src/seedu/addressbook/ui/Formatter.java

```diff
@@ -24,6 +25,17 @@
     /** Offset required to convert between 1-indexing and 0-indexing.  */
     private static final int DISPLAYED_INDEX_OFFSET = 1;
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public static String getPrintableString(Printable... printables) {
+        StringBuilder sb = new StringBuilder();
+        for (Printable printable : printables) {
+            sb.append(printable.getPrintableString());
+            sb.append(" ");
+        }
+        return sb.toString();
+    }
```

Comment:
Nice solution :)

An alternative would be to use Java's `StringJoiner` instead.